packages/babel-plugin-transform-no-undefined-identifier/src/index.js

```diff
@@ -0,0 +1,27 @@
+export default function() {
+  return {
+    visitor: {
+      CallExpression(path, state) {
+        const args = path.get("arguments");
+        const evaluatedCallee = path.get("callee").evaluate();
```

Comment:
These lines should be moved after having checked if the file needs to be linted (lines 8-10), so we avoid evaluating the callee when it isn't needed.

Reply:
:man_facepalming: nice catch!

Reply:
think i might remove the filename check from this file altogether. seems more appropriate to do that in the makefile command to me